platform-api/src/main/java/io/datacater/core/config/ConfigEntity.java

```diff
@@ -0,0 +1,93 @@
+package io.datacater.core.config;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.datacater.core.config.enums.Kind;
+import io.datacater.core.exceptions.JsonNotParsableException;
+import io.quarkiverse.hibernate.types.json.JsonBinaryType;
+import io.quarkiverse.hibernate.types.json.JsonTypes;
+import java.util.Date;
+import java.util.Map;
+import java.util.UUID;
+import javax.persistence.*;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.Type;
+import org.hibernate.annotations.TypeDef;
+import org.hibernate.annotations.UpdateTimestamp;
+
+@TypeDef(name = JsonTypes.JSON_BIN, typeClass = JsonBinaryType.class)
+@Table(name = "configs")
+@Entity
+public class ConfigEntity {
+
+  @Id
+  @GeneratedValue
+  @JsonProperty("uuid")
+  private UUID id;
+
+  @CreationTimestamp
+  @JsonProperty("createdAt")
+  private Date createdAt;
+
+  @UpdateTimestamp
+  @JsonProperty("updatedAt")
+  private Date updatedAt;
+
+  @JsonProperty("name")
+  private String name;
+
+  @JsonProperty("kind")
+  private String kind;
+
+  @Type(type = JsonTypes.JSON)
+  @JsonProperty("config")
+  @Column(name = "config", columnDefinition = JsonTypes.JSON_BIN)
+  private JsonNode config;
```

Comment:
We might want to align these with the other resources, and call it `spec`.

Reply:
I pushed changes that implement your request